lib/gtfs.ex

```diff
@@ -69,6 +84,18 @@ defmodule Gtfs do
   # Queries (Server)
 
   @impl true
+  def handle_call(:ready, _from, state) do
+    case state do
+      {:loaded, _gtfs_data} ->
+        {:reply, true, state}
+
+      # This shouldn't happen,
+      # The handle_continue means the server loads before accepting requests
+      :not_loaded ->
+        {:reply, false, state}
```

Comment:
Saying "This shouldn't happen" in a comment is a big smell. That could be avoided by this
```elixir
  def handle_call(:ready, _from, {:loaded, _gtfs_data} = state) do
      {:reply, true, state}
  end
```
Not having a function case for this is a more normal way to say that it shouldn't happen, but it would feel weird for this function, whose only job is to tell you if it's loaded, not handle the `:not_loaded` case.